Libraries/StyleSheet/ColorValueTypes.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
If ProcessedColorValue goes to processColor.js, and ColorValue stays in StyleSheet, then no product code would ever need to import ColorValueTypes.js

Keeping this file as an implementation details sounds great to me :)

Reply:
Ok, I put the export of ProcessedColorValue in processColor.js and the export of ColorValue in StyleSheetTypes.js.   There's really no need for this file anymore so I just deleted it.